addons/website_mass_mailing/static/src/js/website_mass_mailing.editor.js

```diff
@@ -104,6 +104,7 @@ options.registry.mailing_list_subscribe = options.Class.extend({
         checkboxEl.setAttribute('string', _t("Display Thanks Button"));
         checkboxEl.dataset.toggleThanksButton = 'true';
         checkboxEl.dataset.noPreview = 'true';
+        checkboxEl.dataset.dependencies = "!form_opt";
```

Comment:
Hello @bso-odoo 
I'm a bit surprised to see this commit merged :grimacing: 
We have an option to display the thanks view of the block, if there is no thanks view, the option should be hidden. But here, the fix you propose is to hide the thanks view when the form template is selected (even if there is a thanks view).

As QSM requested at the beginning [here](https://github.com/odoo/odoo/pull/91539#issuecomment-1134552513), it's useful for the users to be able to edit the thanks button. After the introduction of the new form view of this block, BVR made [a fix](https://github.com/odoo/odoo/commit/ccf338c67219b10fe86b15012e143a84c886654f) to make the display thanks view working with it.

We also have [a task](https://www.odoo.com/web#id=3645991&cids=1&menu_id=4720&action=333&active_id=974&model=project.task&view_type=form) to be able to edit those thanks views in translate mode.

IMO the fix here should be to hide the option only when there is no thanks view to display.

CC @rdeodoo